index.html

```diff
@@ -143,7 +143,109 @@ <h4 class="heading heading_level-4">Подушка</h4>
                 </div>
             </div>
         </section>
-
+        <section class="section__services">
+            <div class="column-wrapper">
+              <div class="columns section__services-services">
+                <div class="column column_s-2 column_m-2 column_l-4">
+                  <div class="service">
+                    <div class="service__img">
+                      <img
+                        src="images/services/support.png"
+                        alt="img"
+                      />
+                    </div>
+                    <p class="service__name">Быстрая доставка по всей России</p>
+                  </div>
+                </div>
+      
+                <div class="column column_s-2 column_m-2 column_l-4">
+                  <div class="service">
+                    <div class="service__img">
+                      <img
+                        src="images/services/return.png"
+                        alt="img"
+                      />
+                    </div>
+                    <p class="service__name">Круглосуточная служба поддержки</p>
+                  </div>
+                </div>
+                <div class="column column_s-2 column_m-2 column_l-4">
+                  <div class="service">
+                    <div class="service__img">
+                      <img
+                        src="images/services/delivery.png"
+                        alt="img"
+                      />
+                    </div>
+                    <p class="service__name">
+                      100% гарантия возврата и подлинность товара
+                    </p>
+                  </div>
+                </div>
+              </div>
+            </div>
+        </section>
+        <footer class="footer">
+            <div class="column-wrapper">
+              <div class="footer__inner">
+                <div class="columns footer__content">
+                  <div class="column column_s-2 column_m-2 column_l-4">
+                    <div class="messangers">
+                      <a class="messanger__messanger"
```

Comment:
Здесь имя элемента не соответствует правилу именования по БЭМ https://ru.bem.info/methodology/naming-convention/
```suggestion
                      <a class="messangers__messanger"
```

Reply:
Исправил опечатку